src/web/utils/textHelpers.spec.ts

```diff
@@ -96,15 +99,39 @@ describe('Text helper tests', () => {
     });
   });
 
-  describe('sortstrings', () => {
+  describe('sort strings alphabetically', () => {
```

Comment:
Is it worth adding tests for other cases - mixed case strings, special characters, duplicate strings, single element, empty array?

Reply:
Added more tests to account for this